setup.py

```diff
@@ -0,0 +1,51 @@
+import os
+import imp
+
+from setuptools import setup, find_packages
+
+with open("README.md") as f:
+    readme = f.read()
+
+
+version_file = os.path.abspath("pyblish_standalone/version.py")
+version_mod = imp.load_source("version", version_file)
+version = version_mod.version
+
+
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 2",
+    "Programming Language :: Python :: 2.6",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.1",
+    "Programming Language :: Python :: 3.2",
+    "Programming Language :: Python :: 3.3",
+    "Programming Language :: Python :: 3.4",
+    "Programming Language :: Python :: 3.5",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: Utilities"
+]
+
+
+setup(
+    name="pyblish-standalone",
+    version=version,
+    description="Plug-in driven automation framework for content",
+    long_description=readme,
+    author="Abstract Factory and Contributors",
+    author_email="marcus@abstractfactory.io",
+    url="https://github.com/pyblish/pyblish-standalone",
+    license="LGPL",
+    packages=find_packages(),
+    zip_safe=False,
+    classifiers=classifiers,
+    package_data={"pyblish_standalone": ["plugins/*.py"]},
+    install_requires=[
+        "pyblish-base>=1.4",
+        "pyblish-qml>=0.7.0"
```

Comment:
ahh, yes that was it.

I agree it shouldn't be dependent on QML, so I'll remove it.